packaging/setup/plugins/ovirt-engine-common/ovirt-engine-keycloak/db/connection.py

```diff
@@ -118,17 +109,11 @@ def _commands(self):
         dbovirtutils.detectCommands()
 
     @plugin.event(
-        stage=plugin.Stages.STAGE_CUSTOMIZATION,
+        stage=plugin.Stages.STAGE_SETUP,
         name=okkcons.Stages.DB_CONNECTION_SETUP,
-        after=(
-            okkcons.Stages.CORE_ENABLE,
-        ),
         condition=lambda self: (
-            self.environment[oengcommcons.KeycloakEnv.ENABLE] and
```

Comment:
Removing this condition means that if there is a conf file `/etc/ovirt-engine/engine.conf.d/12-setup-keycloak.conf` and in it a non-empty `KEYCLOAK_DB_PASSWORD` _but_ we fail to connect to the database using the credentials there, it will fail engine-setup. Not a big issue IMO, and this is also how it works with all other components (e.g. DWH).

Reply:
Makes sense